test/servinge2e/servicemesh_test.go

```diff
@@ -18,6 +34,18 @@ const (
 	serviceMeshTestProxyImage    = "registry.svc.ci.openshift.org/openshift/knative-v0.15.2:knative-serving-test-httpproxy"
 )
 
+func assumeServiceMeshInstalled(ctx *test.Context) {
+	// Verify the serviceMeshTestNamespace is part of the mesh
+	namespace, err := ctx.Clients.Kube.CoreV1().Namespaces().Get(serviceMeshTestNamespaceName, meta.GetOptions{})
+	if err != nil {
+		ctx.T.Fatalf("failed to verify %q namespace labels: %v", serviceMeshTestNamespaceName, err)
```

Comment:
AFAIK, whenever we call `Fatalf` or `Fatal` we start the message with a capital letter. That's how it's done in most tests in this repo.

Reply:
is that really our convention? I thought all those cases are just violations of the https://github.com/golang/go/wiki/CodeReviewComments#error-strings convention...

Reply:
I see a little difference between using `fmt.Error` and `t.Fatal` . The strings printed through `t.Fatal` are usually (always?) at the beginning right after the test name: `blue_green_test.go:163: Error sending requests: ....` 
Everywhere I look in Knative there's a capital letter used with `t.Fatal`

Reply:
Knative use capital letter in t.Errorf as well...  I don't think a convention where Errorf => lowercase , Fatalf => capital case  would make sense... we should either follow https://github.com/golang/go/wiki/CodeReviewComments#error-strings  (all starting lowercase)  ,  or follow the Knative convention (all capital case)     ( I don't care either way, just want to have it consistent)

Reply:
OK, never mind,  apparently those cases in Knative are just outliers...

Reply:
I see your point of Fatal always appearing at the beginning...

Reply:
ok, capitalized all Fatalf messages..

Reply:
Thank you